starknet_devnet/blueprints/postman.py

```diff
@@ -55,3 +57,39 @@ async def flush():
 
     result_dict = await state.starknet_wrapper.postman_flush()
     return jsonify(result_dict)
+
+
+@postman.route("/send_message_to_l2", methods=["POST"])
+async def send_message_to_l2():
+    """L1 to L2 message mock endpoint"""
+    request_json = request.json or {}
+
+    # Generate transactions
+    transaction = InternalL1Handler.create(
+        contract_address=hex_converter(request_json, "l2_contract_address"),
+        entry_point_selector=hex_converter(request_json, "entry_point_selector"),
+        calldata=[
+            hex_converter(request_json, "l1_contract_address"),
+            *hex_converter(request_json, "payload", to_int_array),
+        ],
+        nonce=hex_converter(request_json, "nonce"),
+        chain_id=state.starknet_wrapper.get_state().general_config.chain_id.value,
+    )
+
+    result = await state.starknet_wrapper.mock_message_to_l2(transaction)
+    return jsonify({"invoke_tx_hash": result})
```

Comment:
Is it really `"invoke_"`? We could make it the same as in gateway.py - there we return a property `"transaction_hash"`

Reply:
https://github.com/Shard-Labs/starknet-devnet/pull/365/commits/0f7272e7f5ea2c140e9b2ae160de4275b4795a7f